.github/workflows/ci.yaml

```diff
@@ -30,25 +30,28 @@ jobs:
   liberica:
     name: ⚡ liberica Check
     runs-on: ubuntu-latest
-    
+
     defaults:
       run:
         working-directory: liberica
-        
+
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
-        
+
       - uses: actions/cache@v3
         with:
           path: liberica/node_modules
           key: ${{ runner.os }}-liberica-mister-x
-        
+
       - name: 🚧 Setup bun
         uses: oven-sh/setup-bun@v1
 
       - name: 🚧 Install dependencies
         run: bun install
 
+      - name: 🔨 Build liberica
+        run: bun run build
```

Comment:
It may be better to use eslint for that, by using `...tseslint.configs.strictTypeChecked` and settings the `parserOptions`

This *should* have the same diagnostics as tsc, but should be faster

See: https://typescript-eslint.io/getting-started/typed-linting/

Reply:
Maybe it is even possible to use (SWC)[https://github.com/swc-project/swc] for faster parsing.